package.json

```diff
@@ -13,7 +13,10 @@
     "test": "babel-node node_modules/.bin/isparta cover _mocha -- --compilers js:babel-register 'src/**/_spec/*.js'",
     "posttest": "istanbul check-coverage --branches 100",
     "testBuild": "mocha buildTest.js",
-    "watch": "mocha --watch --compilers js:babel-register 'src/**/_spec/*.js'"
+    "watch": "mocha --watch --compilers js:babel-register 'src/**/_spec/*.js'",
+    "docs:prepare": "gitbook install",
```

Comment:
You can view the gitbook locally by running `npm run docs:watch`
The docs are hosted as a github page on the branch `gh-pages`